google-cloud-bigtable-bom/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-bigtable-bom</artifactId>
+    <version>1.6.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigtable:current} -->
+    <packaging>pom</packaging>
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-shared-config</artifactId>
+        <version>0.2.1</version>
+    </parent>
```

Comment:
I'm not sure if this is right.  This bom is meant to be imported by end users, but the shared config includes plugin configs  that are meant to be used by client developers

Reply:
Unless end users are using this artifact as their parent, they won't inherit the build rules in `google-cloud-shared-config`. We use that parent for all the release configuration (no compile/test/runtime dependency management in there)

Reply:
Looks like other projects are using this pattern:
https://github.com/jersey/jersey/blob/faa809da43538ce31076b50f969b4bd64caa5ac9/bom/pom.xml#L48-L53
https://github.com/sonatype/oss-parents/blob/5cc24e62cae0fc8a1a6a917c581574b71dec476f/jvnet-parent/pom.xml